internal/view/live_view.go

```diff
@@ -78,6 +78,7 @@ func (v *LiveView) Init(_ context.Context) error {
 
 	v.app.Styles.AddListener(v)
 	v.StylesChanged(v.app.Styles)
+	v.setFullScreen(v.app.Config.K9s.UI.FullScreenLView)
```

Comment:
If we offer this for liveView then detailsView should accept this option as well.